crates/bevy_pbr/src/prepass/prepass.wgsl

```diff
@@ -43,7 +43,8 @@ fn vertex(vertex_no_morph: Vertex) -> VertexOutput {
 #endif
 
 #ifdef SKINNED
-    var model = skinning::skin_model(vertex.joint_indices, vertex.joint_weights);
+    let skin_index = mesh[get_instance_index(vertex_no_morph.instance_index)].skin_index;
```

Comment:
Didn't we get rid of get_instance_index() after upgrading to wgpu 0.19? 🤔

Reply:
Yup.